packages/desktop-client/src/components/reports/chart-theme.js

```diff
@@ -1,10 +1,10 @@
 import { colors } from '../../style';
 
 let colorFades = {
-  blueFadeStart: 'rgba(229, 245, 255, 1)',
-  blueFadeEnd: 'rgba(229, 245, 255, 0)',
-  redFadeStart: 'rgba(255, 243, 242, 1)',
-  redFadeEnd: 'rgba(255, 243, 242, 0)',
+  blueFadeStart: 'rgba(44, 192, 144, 0.3)',
+  blueFadeEnd: 'rgba(44, 192, 144, 0)',
+  redFadeStart: 'rgba(172, 115, 235, 0.3)',
+  redFadeEnd: 'rgba(172, 115, 235, 0)',
```

Comment:
Where are these coming from? Could they be pulled directly from the theme?

Reply:
I couldn't find a way to easily merge my colors, which are hex, into an rgba field. These work for both light and dark and one of the vary few places colors are still hard coded.

Reply:
Oh right, hadn’t noticed the alpha channel. That is one of the limitations of CSS variables at the moment so probably fine to keep hard coded. Maybe add comments saying what palette colors these are based on?

Reply:
Note added.